src/database.py

```diff
@@ -1,23 +1,25 @@
-import os
-from supabase import create_client, Client
-from dotenv import load_dotenv
-
-load_dotenv()
-
-def get_client():
-    url: str = os.environ.get("host")
-    password: str = os.environ.get("password")
-    supabase: Client = create_client(url, password)
-    return supabase
-
-db = get_client()
-
-if __name__ == "__main__":
-    print(
-        db.table('subsections')
-        .select("keyphrases")
-        .eq('section_id', '04-02')
-        .eq('subsection', 1)
-        .execute()
-        .data
-        )
\ No newline at end of file
+import os
+from supabase import create_client, Client
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def get_client():
+    url: str = os.getenv("host")
+    password: str = os.getenv("password")
+    supabase: Client = create_client(url, password)
+    return supabase
+
+
+db = get_client()
+
+if __name__ == "__main__":
+    print(
+        db.table("subsections")
+        .select("keyphrases")
+        .eq("section_id", "04-02")
+        .eq("subsection", 1)
+        .execute()
+        .data
+    )

```

Comment:
Since we're looking into changing section_id (from str to int), might be better to not to include them in the test query. Printing the first output could be fine?

```suggestion
print(db.table("subsections")
        .select("keyphrases")
        .data[0])
```